atom/common/native_mate_converters/blink_converter.cc

```diff
@@ -384,4 +385,33 @@ v8::Local<v8::Value> MediaFlagsToV8(v8::Isolate* isolate, int mediaFlags) {
   return mate::ConvertToV8(isolate, dict);
 }
 
+v8::Local<v8::Value> Converter<blink::WebCache::ResourceTypeStat>::ToV8(
+    v8::Isolate* isolate, 
+    const blink::WebCache::ResourceTypeStat& stat) {
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
+  
+  dict.Set("count", (uint32_t)stat.count);
+  dict.Set("size", (uint32_t)(stat.size >> 10));
```

Comment:
We can probably convert it to double, so the worst case is we lose some precision.